mission_yaml/SEA45_M48.yml

```diff
@@ -373,11 +448,10 @@ profile_variables:
 
   instrument_ctd:
     comment:    unpumped RBR legato CTD
-    calibration_date:     "2021-03-01"
-    calibration_report:  " "
+    calibration_date:     "2020-07-29"
     factory_calibrated:  "yes"
-    long_name:           RBR legato seaexplorer CTD
-    make_model:          RBR legato
+    long_name: RBR legato CTD
+    make_model:  RBR legato CTD
     platform:            platform
-    serial_number:       "206523"
-    type:                platform
+    serial_number:       "205048"
+    type:                platform
```

Comment:
There is a 'no newline at end of file' warning. Is this a problem?

Reply:
Not really an issue, but not very neat either. I've added one